src/redux/cofigureStore.js

```diff
@@ -0,0 +1,18 @@
+import { combineReducers } from 'redux';
+import { configureStore } from '@reduxjs/toolkit';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
```

Comment:
[Optional Changes]: redux-thunk came with redux-toolkit(RTK), you may want to consider refactoring 
Thank you so much, you done a good job 👍